build.sbt

```diff
@@ -2,20 +2,22 @@ name := "millfork"
 
 version := "0.3.23-SNAPSHOT"
 
-scalaVersion := "2.12.11"
+scalaVersion := "2.12.12"
 
 resolvers += Resolver.mavenLocal
 
 libraryDependencies += "com.lihaoyi" %% "fastparse" % "1.0.0"
 
 libraryDependencies += "org.apache.commons" % "commons-configuration2" % "2.2"
 
-libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.8" % "test"
+libraryDependencies += "org.eclipse.lsp4j" % "org.eclipse.lsp4j" % "0.9.0"
 
-libraryDependencies += "com.codingrodent.microprocessor" % "Z80Processor" % "2.0.2" % "test"
+// libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.8" % "test"
```

Comment:
I need to revert these comments (and we need to figure out how to dynamically ignore these).